src/api/filesystem.go

```diff
@@ -0,0 +1,181 @@
+package api
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+	"strings"
+
+	wh "github.com/skycoin/skycoin/src/util/http"
+)
+
+// saveDataRequest
+type saveDataRequest struct {
+	Filename string                 `json:"filename"`
+	Data     map[string]interface{} `json:"data"`
+	Update   bool                   `json:"update"`
+}
+
+// Save arbitrary data to disk
+// URI: /api/v2/data
+// Method: POST
+// Args:
+//     filename: filename [required]
```

Comment:
Filename should not be part of the API, use a single known filename, like data.db uses. Consider this another database file